### PR TITLE
[CBRD-24117] Revised test case for separating multiple SQL statements

### PR DESCRIPTION
**Pull request by zionyun**
refer to http://jira.cubrid.org/browse/CBRD-24117

### CODE REVIEW

File: sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql [hunk 1]
```diff
@@ -4,7 +4,7 @@ set names binary;
 -- constant
 select 1,collation('8'),coercibility('8'),collation('¥' collate binary),coercibility('¥' collate binary) from db_class where '8' > '¥' collate utf8_ja_exp limit 1;
 select 1,collation(('8' > '¥') collate binary),coercibility(('8' > '¥') collate binary) from db_class where ('8' > '¥') collate binary limit 1;
-select coercibility(select _binary'a' union select _binary'b');
+select coercibility("select _binary'a' union select _binary'b'");
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql]: Why is it in quotation marks?
This seems to be irrelevant to the concept of this change.

File: sql/_27_banana_qa/issue_12401_DML_Natural_Join/cases/03_natural_join_bug_from_web_001.sql [hunk 2]
```diff
@@ -52,7 +52,7 @@ select * from a,b where a.id=b.id and a.name=b.name order by 1,2;
 
 select count(*) from a natural join b;
 
-select * from a  minus  order by 1,2 select * from b order by 1,2;
+select * from a  minus  order by 1,2; select * from b order by 1,2;
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12401_DML_Natural_Join/cases/03_natural_join_bug_from_web_001.sql]: This was often the case, but it would be better to use a newline based on the delimiter.

```
select * from a minus order by 1,2; 
select * from b order by 1,2;
```

File: sql/_29_CTE_recursive/_07_supplement/cases/01_cte_mix_common.sql [hunk 3]
```diff
@@ -10,7 +10,7 @@ UNION all
 select e.empId , e.fname , e.sname , e.bossid
 FROM t e inner JOIN cte r ON ( e.bossId= r.a))
 SELECT c,b
-FROM cte SELECT *
+FROM cte; SELECT *
```
Comment (ctshim) [inline sql/_29_CTE_recursive/_07_supplement/cases/01_cte_mix_common.sql]: It's better to put a newline character to separate it into two lines.

```
SELECT c,b FROM cte; 
SELECT * FROM t order by 1;
```

**Comment by ctshim** [comment]
In the file below, after prepare / execute, it is a structure to perform prepare again without performing "DEALLOCATE PREPARE st;".
Shouldn't the scenario be modified?

`sql/_12_mysql_compatibility/_11_code_coverage/_02_other_functions/cases/_002_host_vars.sql`

File: sql/_13_issues/_12_2h/cases/bug_bts_10105.sql [hunk 4]
```diff
@@ -10,7 +10,7 @@ select * from t1;
 
 drop table if exists t1,t2;
 
-create table t2 as SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A  SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A  ;;
+create table t2 as SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A  SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A ;
```
Comment (ctshim) [inline sql/_13_issues/_12_2h/cases/bug_bts_10105.sql]: It should be divided into two sentences as follows.

create table t2 as SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A;
SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A ;

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 5]
```diff
@@ -44,26 +44,26 @@ drop class if exists "select";
...
+desc backupfiles;
 INSERT INTO backupfiles VALUES ('$archive_file_name', '$archiveID', '$CURRENTVAULT', '$checkSum', CURDATE());
 select count(*) from backupfiles;
 DROP TABLE IF EXISTS backupfiles;
 
 DROP TABLE if exists t1;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-CREATE TABLE IF NOT EXISTS t1 SELECT * FROM t1 order by 1;
+CREATE TABLE IF NOT EXISTS t1; SELECT * FROM t1 order by 1;
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: CREATE TABLE IF NOT EXISTS t1 AS SELECT * FROM t1 order by 1;

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 6]
```diff
@@ -75,14 +75,14 @@ DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
 create table t2 (i int);
-create table if not exists t2  (select * from t1);
+create table if not exists t2; (select * from t1);
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: create table if not exists t2 AS (select * from t1);

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 7]
```diff
@@ -75,14 +75,14 @@ DROP TABLE if exists t1,t2;
...
 create table if not exists t2 as (select * from t1);
-create table if not exists t2 (select * from (select * from t1 union select * from t1 ) tt);
+create table if not exists t2; (select * from (select * from t1 union select * from t1 ) tt);
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: create table if not exists t2 AS (select * from (select * from t1 union select * from t1 ) tt);

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 8]
```diff
@@ -91,13 +91,13 @@ create table t2 as (select * from t1);
 DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-create table if not exists t2 (select * from (select * from t1 union select * from t1 ) tt);
+create table if not exists t2; (select * from (select * from t1 union select * from t1 ) tt);
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: create table if not exists t2 AS (select * from (select * from t1 union select * from t1 ) tt);

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 9]
```diff
@@ -111,26 +111,26 @@ select * from t2 order by 1;
 DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-CREATE TABLE IF NOT EXISTS t2 SELECT * FROM t1;
+CREATE TABLE IF NOT EXISTS t2; SELECT * FROM t1;
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: CREATE TABLE IF NOT EXISTS t2 AS SELECT * FROM t1;

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 10]
```diff
@@ -111,26 +111,26 @@ select * from t2 order by 1;
...
 
 DROP TABLE if exists t1,t2;
 
 DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-CREATE TABLE IF NOT EXISTS t1 SELECT * FROM t1;
+CREATE TABLE IF NOT EXISTS t1; SELECT * FROM t1;
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: CREATE TABLE IF NOT EXISTS t1 AS  SELECT * FROM t1;

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql [hunk 11]
```diff
@@ -111,26 +111,26 @@ select * from t2 order by 1;
...
 
 DROP TABLE if exists t1,t2;
 
 DROP TABLE IF EXISTS ttt;
-CREATE TABLE IF NOT EXISTS ttt SELECT 0 AS n;
-CREATE TABLE IF NOT EXISTS ttt SELECT 0 AS n;
+CREATE TABLE IF NOT EXISTS ttt; SELECT 0 AS n;
+CREATE TABLE IF NOT EXISTS ttt; SELECT 0 AS n;
```
Comment (ctshim) [inline sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: CREATE TABLE IF NOT EXISTS ttt AS SELECT 0 AS n;
CREATE TABLE IF NOT EXISTS ttt AS SELECT 0 AS n;

**Comment by ctshim** [comment]
In the bug_bts_6546.sql file, the syntax below will be error-processed, so it would be better to correct the annotation.

```
--this sql will return two values as missing 'from'
select 1 (select * from (select 2) b);
```

File: sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql (hunk 1 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql]: If I do not enclose it in double quotation marks, the syntax error is not resolved.

File: sql/_27_banana_qa/issue_12401_DML_Natural_Join/cases/03_natural_join_bug_from_web_001.sql (hunk 2 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12401_DML_Natural_Join/cases/03_natural_join_bug_from_web_001.sql]: The review has been reflected.

File: sql/_29_CTE_recursive/_07_supplement/cases/01_cte_mix_common.sql (hunk 3 above)
Comment (zionyun) [reply sql/_29_CTE_recursive/_07_supplement/cases/01_cte_mix_common.sql]: The review has been reflected.

File: sql/_13_issues/_12_2h/cases/bug_bts_10105.sql (hunk 4 above)
Comment (zionyun) [reply sql/_13_issues/_12_2h/cases/bug_bts_10105.sql]: This review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 6 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql:78]: The review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 7 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql:80]: The review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 8 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: The review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 6 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: This review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 7 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: This review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 9 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: This review has been reflected.

File: sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql (hunk 11 above)
Comment (zionyun) [reply sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql]: This review has been reflected.

**Comment by zionyun** [comment]
> In the bug_bts_6546.sql file, the syntax below will be error-processed, so it would be better to correct the annotation.
> 
> ```
> --this sql will return two values as missing 'from'
> select 1 (select * from (select 2) b);
> ```

File: sql/_13_issues/_10_1h/cases/bug_2894.sql [hunk 12]
```diff
@@ -1,83 +1,83 @@
...
 
-prepare s from 'select 1 from db_root where ?<>0 and ?<>0' 
-execute s using true, false
+prepare s from 'select 1 from db_root where ?<>0 and ?<>0';
+execute s using true, false;
 deallocate prepare s;
 
-prepare s from 'select 1 from db_root where ?<>0 and ?<>0' 
+prepare s from 'select 1 from db_root where ?<>0 and ?<>0';
 execute s using false, true
```
Comment (ctshim) [inline sql/_13_issues/_10_1h/cases/bug_2894.sql]: You need a semicolon at the end of the line.
`execute s using false, true;`

Comment (zionyun) [reply sql/_13_issues/_10_1h/cases/bug_2894.sql]: This review has been reflected.

File: sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql (hunk 1 above)
Comment (ctshim) [reply sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql]: Please do not correct the query and change the error code on the answer sheet.

Comment (zionyun) [reply sql/_27_banana_qa/issue_16066_BINARY_charset/_02_coercibility/cases/_09_constant_03.sql]: ok, I checked it.

File: medium/_01_fixed/cases/clshar.sql [hunk 13]
```diff
@@ -1,6 +1,6 @@
 autocommit off;
 create class cc 
-class attribute(ca int default 10) 
+class attribute(ca int default 10);
 (a int, b int shared 123, class cb int default 20);
 insert into cc values( 99, 123 )
```
Comment (ctshim) [inline medium/_01_fixed/cases/clshar.sql]: retry check, please.

```
class attribute(ca int default 10)
(a int, b int shared 123, class cb int default 20);
insert into cc values( 99, 123 );
```

Comment (zionyun) [reply medium/_01_fixed/cases/clshar.sql]: ok, this review has been reflected.

**Review by ctshim — APPROVED** [review]
